<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto Final | Regresión lineal | Polinomios de Lagrange </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>841</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inData">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar datos </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Solver">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagrange</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="NumData">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de datos</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValoX">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X desde:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValoY">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Y desde:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interY">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>200</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="interX">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="ValoX_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasta:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interX_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="ValoY_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasta:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interY_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Intervalo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saltos">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Output">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>590</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresión Lineal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Absorbancia:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Abs">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAbs">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Concentración</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
